### PR TITLE
Support pickling UUID objects

### PR DESCRIPTION
**Pull request by lepenkinya**
Pickle is not yet supported, here is a PR which make uuid object pickable.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -284,6 +280,11 @@ impl UUID {
             uuid: Uuid::from_u128(int),
         })
     }
+
+    fn __reduce__<'a>(&self, py: Python<'a>) -> PyResult<&'a PyTuple> {
```
Comment (aminalaee) [inline src/lib.rs:284]: I think it’s simpler to use ‘__getnewargs__’ as is explained here:
https://github.com/PyO3/pyo3/issues/100#issuecomment-1588641927

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "uuid-utils"
-version = "0.3.0"
```
Comment (aminalaee) [inline Cargo.toml:3]: Can we do this separately?

**Review by aminalaee — COMMENTED** [review]
Thanks for the PR. A small comment.

File: Cargo.toml (hunk 2 above)
Comment (lepenkinya) [reply Cargo.toml]: Sure

File: src/lib.rs (hunk 1 above)
Comment (lepenkinya) [reply src/lib.rs]: Thx, improved that one

**Review by aminalaee — APPROVED** [review]
Thanks for the PR :+1:

**Pull request merged**